pkgs/by-name/cy/cyme/package.nix

```diff
@@ -13,16 +13,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cyme";
-  version = "1.8.4";
+  version = "2.1.0";
 
   src = fetchFromGitHub {
     owner = "tuna-f1sh";
     repo = "cyme";
     rev = "v${version}";
-    hash = "sha256-5433lq3u+s4LiC9089Ul7wGJiouQdVDoM3RT0QSiAnU=";
+    hash = "sha256-1UjzTf0lX4WKNrNR2n/Xb0sfFvF/OnGbHtx6FP3SlM0=";
   };
 
-  cargoHash = "sha256-EW4M072qWCghg4UlhjMBR6DVzKsu/foE+j4MOSiHqNk=";
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "nusb-0.1.10" = "sha256-fo4Bi82/4tlYCeunpFrLVV3J2FtF+P95mF8JwTzVVbA=";
```

Comment:
Hmm... they looks waiting the officially releasing 2.x (GitHub releases and homebrew are bumped)

https://github.com/tuna-f1sh/cyme/issues/40

So after releasing https://github.com/kevinmehall/nusb/pull/85 and applied in cyme, this `cargoLock` is needless.